app/helpers/adoptable_unique_species_for_tenant_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AdoptableUniqueSpeciesForTenantHelper
```

Comment:
Ah I hate to be a pain in the butt, but can you add a comment here to state why we do this? I.e. this is used in the navbar and therefore it is not scoped to a tenant, so we have to set the tenant before making the query.